onnxruntime/test/util/compare_ortvalue.cc

```diff
@@ -168,15 +223,16 @@ std::pair<COMPARE_RESULT, std::string> CompareBFloat16Result(const Tensor& outva
 
 std::pair<COMPARE_RESULT, std::string> CompareTwoTensors(const Tensor& outvalue, const Tensor& expected_tensor,
                                                          double per_sample_tolerance,
-                                                         double relative_per_sample_tolerance, bool post_processing) {
+                                                         double relative_per_sample_tolerance, bool post_processing,
+                                                         bool dropout_test = false) {
```

Comment:
This is a general purposed utility function. Introducing a flag just to make it support a single special case, Dropout, is not the best idea...

Reply:
Are you ok it I passed an optional function pointer to do the comparison and set result in this function? It will be a little cleaner and if no function pointer is passed then it can apple default element by element comparison